src/app/onion/dashboard/dashboard.component.ts

```diff
@@ -36,6 +60,21 @@ export class DashboardComponent {
     this.activeIndex++;
   }
 
+  /**
+   * Applys status filters to the draft learning objects list
+   * @param filters
+   */
+  async applyFilters(filters: any) {
+    const filter = Array.from(filters.keys());
+    if (filter.length !== 0) {
+      this.workingLearningObjects = await this.getDraftLearningObjects({status: filter});
+    } else {
+      this.workingLearningObjects = await this.getDraftLearningObjects(
+        {status: ['unreleased', 'proofing', 'review', 'rejected', 'waiting']}
```

Comment:
What was the reasoning behind filtering draft Learning Objects by all statuses instead of providing no filters if none are specified by the user?

By default, when the `draftsOnly` filter is specified, that included Learning Objects in all of the statuses listed.